lte/protos/s1ap_service.proto

```diff
@@ -19,10 +19,11 @@ package magma.lte;
 
 option go_package = "magma/lte/cloud/go/protos";
 
-message EnbConnectedResult {
-    repeated uint32 enb_ids = 1;
+message EnbStateResult {
+    // enb ids -> num of UEs connected
+    map<uint32, uint32> enb_state_map = 1;
 }
 
 service S1apService {
-    rpc GetEnbConnected (magma.orc8r.Void) returns (EnbConnectedResult) {}
+    rpc GetEnbState (magma.orc8r.Void) returns (EnbStateResult) {}
```

Comment:
1. Always include comment describing what the rpc does

2. Request and return messages should always be specific to this method, and named as `MethodnameRequest`, `MethodnameResponse` [0]. Here this would be (with consistent casing [1])

```
rpc GetENBState (GetENBStateRequest) returns (GetENBStateResponse) {}
```

[0] https://github.com/uber/prototool/blob/dev/style/README.md#servicesrpcs
[1] https://github.com/golang/go/wiki/CodeReviewComments#initialisms